src/data/move.ts

```diff
@@ -581,11 +584,17 @@ export abstract class Move implements Localizable {
   /**
    * Sets the {@linkcode MoveFlags.G_MAX_MOVE} for the move
    * and {@linkcode moveTarget} to NEAR_ENEMY (g-max moves cannot target allies)
+   * and make the move not make contact
+   * and have the {@linkcode UseHigherAttackingStatAttr} to use the higher of the attacking stat
+   * and have the move use the {@linkcode GMaxPowerAttr}
    * @returns The {@linkcode Move} that called this function
    */
```

Comment:
NIT: and, and and and......and and... phew..
```suggestion
/**
 * Modifies the move with the following properties:
 * - Sets the {@linkcode MoveFlags.G_MAX_MOVE}.
 * - Sets {@linkcode moveTarget} to NEAR_ENEMY (G-Max moves cannot target allies).
 * - Prevents the move from making contact.
 * - Applies {@linkcode UseHigherAttackingStatAttr} to use the higher attacking stat.
 * - Assigns the move the {@linkcode GMaxPowerAttr}.
 * 
 * @returns The {@linkcode Move} that called this function.
 */
```